mangaki/mangaki/management/commands/tokens.py

```diff
@@ -0,0 +1,23 @@
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from mangaki.views import compute_token
+from django.contrib.auth import get_user_model
+from django.db import connection
+
+
+class Command(BaseCommand):
+    args = ''
+    help = 'Generate tokens from HASH_NACL'
```

Comment:
If we want this to be generic; it should take the salt as argument. However, I'm not sure why we don't have a management command for sending out a templated email to users (that template should contain a token) instead.